dojo/tools/burp/parser.py

```diff
@@ -234,7 +234,7 @@ def get_item(item_node, test):
     cwes = do_clean_cwe(item_node.findall('vulnerabilityClassifications'))
     if len(cwes) > 1:
         # FIXME support more than one CWE
-        logger.warning(f"more than one CWE for a finding {cwes}. NOT supported by parser API")
+        logger.debug(f"more than one CWE for a finding {cwes}. NOT supported by parser API")
```

Comment:
Why do you suppress this message? Might be interesting for a production environment to see why there is no CWE in a finding.

Reply:
No. It's not what you're thinking. I created this log to notify that the data in the report have more than one CWE but the current data structure only supports one.
But many many parsers have this design problem. It doesn't make sense to flood the logs with these messages because our implementation is not perfect.
Also, the parsers that generate these warnings use the first one when there is more than one CWE.

Reply:
I really think that we should modify our implementation of Finding to support more than zero or only one CWE.
Maybe an array.